client/Mac/mf_client.m

```diff
@@ -143,6 +143,80 @@ void mf_scale_mouse_event(void* context, rdpInput* input, UINT16 flags,
 	}
 }
 
+void mf_scale_mouse_event_ex(void* context, rdpInput* input, UINT16 flags,
```

Comment:
The code in this function is a bit hard to understand ...
I.e. I don't really get why we only check if the height is equal [```if (wh != dh)```] but don't do the same check for the width?